cms/templates/import.html

```diff
@@ -143,12 +214,13 @@ <h3 class="title-3">${_("Warning: Importing while a course is running")}</h3>
         <p>${_("If you perform an import while your course is running, and you change the URL names (or url_name nodes) of any Problem components, the student data associated with those Problem components may be lost. This data includes students' problem scores.")}</p>
```

Comment:
@Kelketek @antoviaque @marcotuts Can we delete this warning completely? It's not relevant to libraries, I don't think. 
Is there any alternative information/warning that we should provide? Problems inside randomized content blocks that come from a library that has been overwritten will be out of sync, is that correct? So perhaps we replace the warning with something about re-syncing problems to get the up-to-date version in the library? Please confirm, thanks!